### PR TITLE
Fix tox conda env spec

### PR DESCRIPTION
**Pull request by dstansby**
Not sure whether this is intended or a bug in `tox`. Either way this should fix running the tox job. I've tested this locally and it works.

### CODE REVIEW

**Review by nabobalis — APPROVED** [review]

**Review by Cadair — APPROVED** [review]
Pretty sure this is "expected behaviour" in tox 3.

File: tox.ini [hunk 1]
```diff
@@ -121,7 +121,7 @@ commands =
     python -c "import sunpy; sunpy.self_test()"
 
 # Requires tox-conda
-[testenv:conda]
+[testenv:py{37,38,39,310}-conda]
```
Comment (nabobalis) [inline tox.ini:124]:
```suggestion
[testenv:py{38,39,310}-conda]
```
Did we drop 3.7?

**Comment by nabobalis** [comment]
The label doesn't seem to do anything?

**Comment by nabobalis** [comment]
Oh its there now after the commit update

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Oh man, I wonder if we want a `skip_conda` flag.

**Pull request merged**